test/Utility/IndexedArray.cpp

```diff
@@ -1,8 +1,8 @@
-#include <gtest/gtest.h>
```

Comment:
TBH I'd prefer to have tests close to the source code.

This way we don't have to maintain a source tree mirror and keep it in sync with the main directory tree, and switching between test / implementation is easier.

Reply:
Also, another argument for keeping them next to the source code is that this way it's clear that tests are 1st class citizens in the repo, and not some code that's somewhere out there that you'll need to know how to find.

In my experience with different codebases, when organised this way, people write more tests in the long run.